### ISSUE TITLE
ZED LiveLink.exe - Entry point not found

### ISSUE THREAD

**Issue by RSN4D** [stereolabs/zed-livelink-plugin#7]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues.
- [X] This issue is not a question, feature request, or anything other than a bug report directly related to this project.

### Description

i followed the steps to make LiveLinkSender from Scratch...
it works as expected when i make it on my desktop PC with Win10, following the steps on github page.
if i do the exact same on my laptop with Win11, i get the following error message.

using ZED2 SDK 3.8
Cuda 11.7

ZED2 camera can be accessed on laptop using the ZED2 explorer.

Laptop is i9 with 3080ti

### Steps to Reproduce

install following Github doc.

### Expected Result

get LiveLinkSender

### Actual Result

Error Code

### ZED Camera model

ZED2

### Environment

```shell
MSI Laptop
I9, 3080TI
Win11 Pro
```

### Anything else?

![ErrorMessage_Zed2_Win11](https://user-images.githubusercontent.com/16054506/206664974-a5a0b2e4-e369-41fe-9670-c3ae86460efe.png)

**Comment by RSN4D** [comment]
Hi, i will try what you suggested...
i also tried building the zed-ue5-main project, same error, when i try to build project with vs2022 (after direct download from github), i get the following message in output log:

1>------ Build started: Project: ZEDSamples, Configuration: Development_Editor x64 ------
1>Running UnrealBuildTool: dotnet "..\..\Engine\Binaries\DotNET\UnrealBuildTool\UnrealBuildTool.dll" ZEDSamplesEditor Win64 Development -Project="D:\UE\Projects_UE5\zed-UE5-main\ZEDSamples.uproject" -WaitMutex -FromMsBuild
1>Log file: C:\Users\Unreal\AppData\Local\UnrealBuildTool\Log.txt
1>Creating makefile for ZEDSamplesEditor (no existing makefile)
1>ZED SDK Minor Version mismatch : found 7 expected 8
1>Unable to instantiate module 'Stereolabs': ZED SDK Minor Version mismatch : found 7 expected 8
1>(referenced via ZEDSamples.uproject -> Stereolabs.uplugin)
1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.MakeFile.Targets(44,5): error MSB3073: The command ""C:\Program Files\Epic Games\UE_5.1\Engine\Build\BatchFiles\Build.bat" ZEDSamplesEditor Win64 Development -Project="D:\UE\Projects_UE5\zed-UE5-main\ZEDSamples.uproject" -WaitMutex -FromMsBuild" exited with code 6.
1>Done building project "ZEDSamples.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========

**Comment by Bvallon-sl** [comment]
Hi,

It means you have the ZED SDK3.7 installed, not 3.8 :

`Unable to instantiate module 'Stereolabs': ZED SDK Minor Version mismatch : found 7 expected 8`

**Comment by RSN4D** [comment]
yes, my honest apologies, i just installed 3.8.2, everything is working...
thank you for your help.
since this is not a bug, just my fault, should i delete this issue (because it is no issue...)?

**Comment by Bvallon-sl** [comment]
No problem.
You can close the issue.
